CE06ISSM/CE06ISSM_D00020_ingest.csv

```diff
@@ -0,0 +1,30 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+# mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/cpm3/syslog/cpm_status.txt,CE06ISSM-MFC31-00-CPMENG000,telemetered,Pending,Incorrectly specified driver and dataset
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/syslog/*.syslog.log,CE06ISSM-MFD35-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+# mi.dataset.driver.vel3d_cd.dcl.vel3d_cd_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/vel3d/*.vel3d.log,CE06ISSM-MFD35-01-VEL3DD000,telemetered,Not Expected,Configured to record internally
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/presf/*.presf.log,CE06ISSM-MFD35-02-PRESFA000,telemetered,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/adcpt/*.adcpt.log,CE06ISSM-MFD35-04-ADCPTM000,telemetered,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/pco2w2/*.pco2w2.log,CE06ISSM-MFD35-05-PCO2WB000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl36/phsen2/*.phsen*.log,CE06ISSM-MFD35-06-PHSEND000,telemetered,Available,
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl37/syslog/*.syslog.log,CE06ISSM-MFD37-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE06ISSM/D00020/cg_data/dcl37/optaa2/*.optaa*.log,CE06ISSM-MFD37-01-OPTAAD000,telemetered,Available,

```

Comment:
No files found in the X00020 directory on the NAS drive.  Maybe these are "Not Expected" for status?

Reply:
Correct, the OPTAA's did not arrive in time for burn-in, so most of those directories are indeed empty. They are now installed and will be telemetering data during the deployment.